### PR TITLE
Enhanced Guides/Customization.md

### PR DESCRIPTION
**Pull request by 0xrahul6**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
docs update
- **Why was this change needed?** (You can also link to an open issue here)
Enhanced the Customization.md
- **Other information**:

### CODE REVIEW

File: docs/pages/Guides/Customising-prompts.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Customizing the Main Prompt
 
-To customize the main prompt for DocsGPT, follow these steps:
+Customizing the main prompt for DocsGPT gives you the ability to tailor the AI's responses to your specific requirements. By modifying the prompt text, you can achieve more accurate and relevant answers. Here's how you can do it:
 
-1. Navigate to `/application/prompts/combine_prompt.txt`.
```
Comment (dartpain) [inline docs/pages/Guides/Customising-prompts.md:5]: Please leave numbered steps

**Review by dartpain — CHANGES_REQUESTED** [review]
Just a few comments

**Comment by 0xrahul6** [comment]
@dartpain commited the suggestions

**Comment by dartpain** [comment]
Looks like 
Screenshot 2023-10-31 at 13 31 55
1. steps are not numbered as they where before 
2. no ` on the filename

**Comment by 0xrahul6** [comment]
> Looks like Screenshot 2023-10-31 at 13 31 55
> 
>     1. steps are not numbered as they where before
> 
>     2. no ` on the filename

Updated

**Review by dartpain — APPROVED** [review]

**Comment by dartpain** [comment]
Thank you!

**Pull request merged**